contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+
+“Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting...We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”
+
+All of this is reflected in Puzzle’s online messaging, with language which directly addresses founders ahead of accountants and emphasizes how the product integrates simply with other tools which founders already use. 
+
+> **Putting it into practice:** [We advise having an approximate structure for user interviews](/blog/how-to-work-out-what-users-need), but leave room to flex. If users go off on a tangent during an interview that could give you vital [insights to their actual feelings](/handbook/product/user-feedback)!
+
+## 3. Build pricing around user segments
+
+Next, you can delve into the quantitative data by looking at how users can be segmented into different groups based on criteria such as size or industry. Puzzle used this information to structure its pricing model, for example. 
+
+“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and a scale plan for companies which are generating revenue.
+
+Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the scale-up plan to AI automation and new processes.
+
+> **Putting it into practice:** If you aren’t using delineated pricing plans then it can still be helpful to think of your broader [pricing principles](/handbook/engineering/feature-pricing). Ask yourself: how do these principles work for your different user segments? 
+
+## 4. Reinforce your ICP with internal teams
+
+As you grow it’s important not to drift from your ICP or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 
+
+“When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our ICP is a company, while others thought it was a person, and there were some different ideas floating around.”
+
+At PostHog, we’ve solved this issue with [a public handbook](/handbook) which documents [our ICP](/handbook/who-we-are-building-for) — but Puzzle took a different approach.  
+
+“We have these all-hands meetings where everybody has to put their hand up and swear that our ICP is a founder and not an accountant,” says Scott. 
```

Comment:
@joethreepwood would love if we could tweak this a bit. It was kind of an offhanded comment so maybe there's a better message we can use here?